openapi.yaml

```diff
@@ -110,33 +110,50 @@ paths:
         '500':
           description: Internal Server Error
 
-  /v1/users:
+  /v1/users/edit:
     patch:
       tags:
         - Users
       summary: Edit User Profile
+      parameters:
```

Comment:
this should be in the auth section https://swagger.io/docs/specification/authentication/